### ISSUE TITLE
Highmaps - scroll not working correctly

### ISSUE THREAD

**Issue by Dezzley** [superman66/vue-highcharts#44]
When using the component with Highmaps, the scroll functionality is not working properly, no matter which options you feed it (following the official Highcharts API of course).
After a little digging, I've found the problem to be in VueHighcharts.vue in the init() functions.
Suggested fix: If you try and initiate a map, use ' new _Highcharts.mapChart(this.$el, this.options) ' inside init().

**Comment by superman66** [comment]
@Dezzley  Do you mean scroll is Drilldown? Like this demo: https://www.highcharts.com/maps/demo/map-drilldown

**Comment by Dezzley** [comment]
@superman66 I was talking about map in the demo: http://chenhuichao.com/vue-highcharts/#/MapChart   where you can't zoom in/out the map properly because it was initiated via Chart constructor instead of mapChart

**Comment by superman66** [comment]
I fixed it `vue2-highcharts@1.2.5`. But in the demo http://chenhuichao.com/vue-highcharts/#/MapChart, if you want to use zoom in/out ,you should add. 
```
        mapNavigation: {
            enabled: true
        },
```
For more detail, you can see https://jsfiddle.net/6qpt4m2a/.
